metaflow/plugins/kfp/tests/flows/resources_flow.py

```diff
@@ -109,6 +115,62 @@ def start(self):
         assert os.environ.get("MF_TAG_METAFLOW_TEST") == "true"
         assert os.environ.get("MF_TAG_TEST_T1") == "true"
 
+        # test accelerator usage
```

Comment:
@talebzeghmi I know we discussed where to put this test on Tuesday, and I mentioned we can create a new integration test in the same structure as `aip-spark-integration-test`. I'm now thinking it might be ok to keep this test here. This is because the `start` step (which is the only step that uses the `p3` instance) never takes more than 30 seconds. So for every run, we pay only about 2.5 cents additional for using the `p3` instance (based on the current on demand pricing of the `p3` instance). By having it here, we also don't need to manage another integration testing project. Let me know your thoughts, but I'm happy to move it wherever it is best suited.

Reply:
How long does the node last for?

Reply:
Just 30 seconds because the P3 node has a taint that doesn't allow a pod scheduled on it unless that pod has a corresponding toleration for that taint. Only the `start` step in the flow is provided with this toleration.